joplin/base/models/janis_page.py

```diff
@@ -165,11 +164,14 @@ def janis_preview_url_end(self, revision=None):
         """
         if revision:
             url_page_type = revision.page.janis_url_page_type
+            global_id = graphene.Node.to_global_id('PageRevisionNode', revision.id)
```

Comment:
@Niicck ran into some errors when making LocationPage a JanisPage (where I think there werent any revisions yet?)
anyways, this seems to do what you were looking to do while avoiding triggering an error if global_id fails for not existing

Reply:
Ah, nope different logic going on.
```
Optional "revision" parameter to get the janis_preview_url for a specific revision
Otherwise, it will return the janis_preview_url for the latest revision
```
Now you've changed it so that the default preview for a page won't preview the latest revision.

This logic will have to be changed back. If there are errors getting a preview_url for LocationPage that doesn't have a revision, it needs to be handled in a different way.

Reply:
Ok, my bad. Good to know. I'll revisit this.

Reply:
@Niicck this should be correct now